app/views/comments/index.html.erb

```diff
@@ -0,0 +1,18 @@
+<div class="container">
+	<div class="d-flex justify-content-center h-100">
+		<div class="card background comments wall">
+			<div class="card-header">
+			</div>
+			<div class="card-body">
+				<% if @comments.empty? %>
+					<h3>No comments Exists</h3>
+				<% else @comments %>

```

Comment:
simple `else` should also work fine here or you can use `elsif @comments.present?`